htdocs/xoops_lib/Xoops/Core/Assets.php

```diff
@@ -96,7 +96,7 @@ class Assets
      * @var string string to identify Assetic filters using instanceof
      */
 
-    private $filterInterface = '\Assetic\Filter\FilterInterface';
+    private $filterInterface = '\\Assetic\\Filter\\FilterInterface';
```

Comment:
Le 24/09/2015 22:45, Richard Griffith a écrit :
> In htdocs/xoops_lib/Xoops/Core/Assets.php
> <https://github.com/XOOPS/XoopsCore/pull/404#discussion_r40370724>:
>
>> @@ -96,7 +96,7 @@ class Assets
>>       * @var string string to identify Assetic filters using instanceof
>>       */
>>
>> -    private $filterInterface = '\Assetic\Filter\FilterInterface';
>> +    private $filterInterface = '\\Assetic\\Filter\\FilterInterface';
>
> That is very badly worded. The examples clarify it:
>
> |// Outputs: You deleted C:\*.*? echo 'You deleted C:\\*.*?'; // Outputs:
> You deleted C:\*.*? echo 'You deleted C:\*.*?'; |
>
> You can use two backslashes, but one will work in all cases /except/
> when the next character is a single quote or a backslash. I.e. to output
> |\'| you would have to specify |\\\'|
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/XOOPS/XoopsCore/pull/404/files#r40370724>.
>


See php.net

To specify a literal single quote, escape it with a backslash (\).
To specify a literal backslash, double it (\\).

If you want to display two backslash \\,
  you have to write 'two bs \\\\ in string' (single quoted)